app/services/validators/session-handler.js

```diff
@@ -0,0 +1,250 @@
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'benefits') {
+      if (!session['dobEncoded'] ||
+          !session['relationship']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'about-the-prisoner') {
+      if (!session['dobEncoded'] ||
+          !session['relationship'] ||
+          !session['benefit']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'about-you') {
+      if (!session['dobEncoded'] ||
+          !session['relationship'] ||
+          !session['benefit'] ||
+          !session['referenceId'] ||
+          !session['decryptedRef']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'future-or-past-visit') {
+      if (!session['referenceId'] ||
+          !session['decryptedRef'] ||
+          !session['claimType']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'journey-information') {
+      if (!session['referenceId'] ||
+          !session['decryptedRef'] ||
+          !session['claimType'] ||
+          !session['advanceOrPast']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'has-escort' ||
+        page === 'about-escort' ||
+        page === 'has-child' ||
+        page === 'about-child' ||
+        page === 'expenses' ||
+        page === 'accommodation' ||
+        page === 'bus' ||
+        page === 'car' ||
+        page === 'car-only' ||
+        page === 'hire' ||
+        page === 'ferry' ||
+        page === 'refreshment' ||
+        page === 'plane' ||
+        page === 'taxi' ||
+        page === 'train' ||
+        page === 'summary' ||
+        page === 'payment-details' ||
+        page === 'declaration') {
+      if (!session['referenceId'] ||
+          !session['decryptedRef'] ||
+          !session['claimType'] ||
+          !session['advanceOrPast'] ||
+          !session['claimId']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'file-upload') {
+      if (!session['decryptedRef'] ||
+          !session['claimId']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'view-claim' ||
+               page === 'your-claims' ||
+               page === 'update-contact-details' ||
+               page === 'check-your-information') {
+      if (!session['dobEncoded'] ||
+          !session['decryptedRef']) {
+        return false
+      } else {
+        return true
+      }
+    }
+
+    if (page === 'same-journey-as-last-claim') {
+      if (!session['claimType'] ||
+          !session['referenceId'] ||
+          !session['decryptedRef'] ||
+          !session['advanceOrPast']) {
+        return false
+      } else {
+        return true
+      }
+    }
+  }
+}
+
+module.exports = function (session, url) {
```

Comment:
change to expose a named functions, like:
```
module.exports = { 
  validateSession: function (session, url) {...
  getErrorPath: function (session, url) {...
}
```
Right now you expose a function which has a property of another function (getErrorPath)